src/bedrock/monitoring.cpp

```diff
@@ -605,6 +612,15 @@ int main(int argc, char* argv[]) {
   YAML::Node conf = YAML::LoadFile("conf/config.yml")["monitoring"];
   string ip = conf["ip"].as<string>();
 
+  YAML::Node conf_thread = YAML::LoadFile("conf/config.yml")["thread"];
+  MEMORY_THREAD_NUM = conf_thread["memory"].as<int>();
+  EBS_THREAD_NUM = conf_thread["ebs"].as<int>();
+
+  YAML::Node conf_replication = YAML::LoadFile("conf/config.yml")["replication"];
```

Comment:
There's no need to call `LoadFile` three times here. We should just call it once and then access the `monitoring`, `thread`, and `replication` nodes individually.